pkg/nsx/services/vpc/builder.go

```diff
@@ -82,7 +81,7 @@ func buildNSXVPC(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, nc v1alpha1.VPC
 		}
 		vpc.Tags = util.BuildBasicTags(cluster, obj, nsObj.UID)
 		vpc.Tags = append(vpc.Tags, model.Tag{
-			Scope: common.String(common.TagScopeVPCManagedBy), Tag: common.String(common.AutoCreatedVPCTagValue)})
+			Scope: common.String(common.TagScopeManagedBy), Tag: common.String(common.AutoCreatedTagValue)})
```

Comment:
{
"scope": "nsx/managed-by",
"tag": "nsx-op"
}  should be added for auto-created subnets. Also consider the upgrade case, for all subnets in subnet store, add this tag if non-exist.